### PR TITLE
Fixes for Python 3

### PR DESCRIPTION
**Pull request by p-l-**

### CODE REVIEW

**Comment by guedou** [comment]
Approved via email =)

I read everything and it's good to be merged. Can you do it ?

**Comment by p-l-** [comment]
Done, thanks @guedou!

File: scapy/layers/bluetooth.py [hunk 1]
```diff
@@ -103,8 +101,7 @@ class L2CAP_CmdHdr(Packet):
     def post_build(self, p, pay):
         p += pay
         if self.len is None:
-            l = len(p)-4
-            p = p[:2]+chr(l&0xff)+chr((l>>8)&0xff)+p[4:]
+            p = p[:2] + struct.pack("<H", len(pay)) + p[4:]
```
Comment (gpotter2) [inline scapy/layers/bluetooth.py]: @p-l- Hey it seems that this was len-4 ?!

File: scapy/layers/bluetooth.py [hunk 2]
```diff
@@ -464,8 +461,7 @@ class HCI_Command_Hdr(Packet):
     def post_build(self, p, pay):
         p += pay
         if self.len is None:
-            l = len(p)-3
-            p = p[:2]+chr(l&0xff)+p[3:]
+            p = p[:2] + struct.pack("B", len(pay)) + p[3:]
```
Comment (gpotter2) [inline scapy/layers/bluetooth.py]: Same

**Pull request merged**